bot/constants.py

```diff
@@ -224,6 +224,10 @@ class Emojis:
     status_dnd = "<:status_dnd:470326272082313216>"
     status_offline = "<:status_offline:470326266537705472>"
 
+
+    stackoverflow_tag = environ.get('stack_tag', "<:stack_tag:*> ")
+    stackoverflow_views = environ.get('stack_views', "<:stack_eye:*>")
```

Comment:
The IDs here will need to be added eventually, yes?

Reply:
Eventually, yes, it would be needed.

Reply:
So [these](https://github.com/python-discord/sir-lancebot/pull/583#issuecomment-857867503) are the icons that you would need added to the server as emojis?

Reply:
Yup, since for the rest of the icons I am using the reddit icons.

Reply:
The emojis have been added, being `stack_tag` and `stack_eye`.